issue-tracker/src/components/newIssuePopup/newIssuePopup.component.jsx

```diff
@@ -1,24 +1,28 @@
-import React, { useState } from "react";
-
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTimes } from "@fortawesome/free-solid-svg-icons";
 import './popup.styles.css';
 import Modal from "../modal/modal.component";
 import Form from "../form/form.component";
 import Input from "../input/input.component";
+import Button from "../button/button.component";
 
 
-const Popup = props => {
+const NewIssuePopup = props => {
```

Comment:
Will need to make something better out of this, maybe trough inheriting from a normal Popup? No idea... @ByRicardoSimoes

Reply:
Don't know if this is possible